extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -43,7 +43,8 @@ define('ext.wikia.adEngine.adContext', [
 	function setContext(newContext) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 54 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)